core/src/epicli/data/common/ansible/playbooks/roles/preflight/defaults/main.yml

```diff
@@ -1,14 +1,118 @@
----
-supported_oses:
-  - name: CentOS
-    version:
-      min: 7.6
-      max: 7.9
-  - name: RedHat
-    version:
-      min: 7.6
-      max: 7.9
-  - name: Ubuntu
-    version:
-      min: 18.04
-      max: 18.04
+---
+supported_oses:
+  - name: CentOS
+    version:
+      min: 7.6
+      max: 7.9
+  - name: RedHat
+    version:
+      min: 7.6
+      max: 7.9
+  - name: Ubuntu
+    version:
+      min: 18.04
+      max: 18.04
+
+unsupported_roles:
+  - distro: Ubuntu
+    arch: x86_64
+    roles: []  # all supported
+  - distro: Ubuntu
+    arch: aarch64
+    roles:
+      - repository
+      - kafka
+      - firewall
+      - image_registry
+      - kubernetes_master
+      - kubernetes_node
+      - helm
+      - logging
+      - elasticsearch_curator
+      - opendistro_for_elasticsearch
+      - elasticsearch
+      - kibana
+      - filebeat
+      - logstash
+      - prometheus
+      - grafana
+      - node_exporter
+      - jmx_exporter
+      - zookeeper
+      - rabbitmq
+      - kafka_exporter
+      - postgresql
+      - postgres_exporter
+      - haproxy
+      - haproxy_exporter
+      - vault
+      - applications
+      - ignite
+  - distro: CentOS
+    arch: x86_64
+    roles: []  # all supported
+  - distro: CentOS
+    arch: aarch64
+    roles:
```

Comment:
Please enable the components we already support inside the ARM branch till now and make sure they install correctly:

      - repository
      - kafka
      - image_registry
      - kubernetes_master
      - kubernetes_node
      - helm
      - zookeeper
      - haproxy
      - vault
      - firewall

Reply:
Also for centos pls just add all roles are going to support as postgress and applications are already there. So missing would be everything involving logging, monitoring and rabbitmq.